corehq/apps/data_analytics/models.py

```diff
@@ -157,3 +157,94 @@ def export_row(self, past_months):
             experienced_threshold=self.experienced_threshold or DEFAULT_EXPERIENCED_THRESHOLD,
             performance_threshold=self.performance_threshold or DEFAULT_PERFORMANCE_THRESHOLD,
         )
+
+
+class DomainMetrics(models.Model):
+
+    domain = models.TextField(unique=True, db_index=True)
+    date_created = models.DateTimeField(auto_now_add=True)
+    last_modified = models.DateTimeField(auto_now=True)
+
+    first_domain_for_creating_user = models.BooleanField()
```

Comment:
I think I would consider `is_first_domain_for_creating_user`, though not tied to it. It's verbose but it's more immediately clear it should be a boolean.

Reply:
467a3663ab8330c27762a5e26ebd33c7ce1d687d